raaghu-elements/src/rds-datepicker/rds-datepicker.stories.tsx

```
@@ -62,14 +69,17 @@ export const Default: Story = {
+        isMandatory: true,
         placeholderText: "Select Date",
         changeIcon: "calendar",
         datePickerStyleType: DatePickerStyleType.Selector,
         layout: DatePickerLayout.Default,
+        isDisabled: false,
+        showClearDate: true,
+        isDefaultDate: false,
     }
 } satisfies Story;
-Default.parameters = { controls: { include: ['state','type','changeIcon','layout', 'showTitle','title','isMandatory','placeholderText','datePickerStyleType'] } };
+Default.parameters = { controls: { include: ['state', 'type', 'changeIcon', 'layout', 'showTitle', 'title', 'isMandatory', 'placeholderText', 'datePickerStyleType', 'isDisabled', 'showClearDate', 'isDefaultDate'] } };
```

ESLint

[error] Strings must use doublequote.

(quotes)